internal/generator/vector/output/syslog/syslog.go

```diff
@@ -145,6 +146,7 @@ func New(id string, o obs.OutputSpec, inputs []string, secrets observability.Sec
 
 	return append(syslogElements,
 		common.NewAcknowledgments(id, strategy),
+		common.NewBatch(id, strategy),
```

Comment:
This has been added, but not implemented.   NewBuffer is used for delivery mode and already exists here.   Should we additionally add maxWrite to tuning, since NewBatch is added here?

Reply:
Actually it should be NewBuffer, not Batch. I shouldn't have actually added this in. The `socket` output doesn't support batch/request parameters that I can see